crates/sputnik/src/report.rs

```diff
@@ -60,3 +64,33 @@ fn write_machine_id(path: &PathBuf) -> Result<Uuid, SputnikError> {
     file.write_all(machine_id.to_string().as_bytes())?;
     Ok(machine_id)
 }
+
+#[cfg(test)]
+mod tests {
+    use super::{get_or_write_machine_id, write_machine_id};
+
+    use fixture::Fixture;
+
+    #[test]
+    fn it_can_write_machine_id() {
```

Comment:
I think this is fine, since the following tests prove this util works, but this test doesn't necessarily check to make sure the id was written, just that the function doesn't `Err`